s2/bq/doc/REFERENCE.md

```diff
@@ -0,0 +1,162 @@
+## Reference
+
+### S2
+
+#### VERSION
+
+{{% bannerNote type="code" %}}
+s2.VERSION()
+{{%/ bannerNote %}}
+
+Returns the current version of the s2 library.
+
+#### GEOJSONBOUNDARY_FROMKEY
+
+{{% bannerNote type="code" %}}
+s2.GEOJSONBOUNDARY_FROMKEY(key STRING) 
+{{%/ bannerNote %}}
+
+Returns a GeoJSON containing the boundary corners of a quadkey.
+
+* `key`: `STRING` quadkey to extract the corners from.
+
+#### GEOJSONBOUNDARY_FROMLONGLAT
+
+{{% bannerNote type="code" %}}
+s2.GEOJSONBOUNDARY_FROMLONGLAT(longitude FLOAT64, latitude FLOAT64, level NUMERIC)
+{{%/ bannerNote %}}
+
+Returns a GeoJSON containing the boundary corners given a longitude, latitude and zoom level.
```

Comment:
Do we really need this function? We should have `long, lat, level -> id`  and `id -> geog (boundary)`. I don't see why we want to maintain `long, lat, level -> geojson boundary`. This isn't meant to match a JS API, but to provide a good SQL API we can build with.